vpr/src/base/constraints_load.cpp

```diff
@@ -0,0 +1,106 @@
+#include "constraints_load.h"
+
+void print_region(FILE* fp, Region region);
+void print_partition(FILE* fp, Partition part);
```

Comment:
if keeping in this file, put static in front of the declarations so that it can't be called outside this file. Probably better to just move to class file though